server/src/functions/verify-token.func.ts

```diff
@@ -17,8 +17,8 @@ export function verifyToken(res: Response, token: string, next: NextFunction): v
                 return res.status(403).send(wrapResponse(false, { error: 'Error occured during authorization!' }));
             }
             const user = await User.scope('full').findByPk(userData.id);
-
-            if (user === null) {
+            // No user found, or User has no organization
+            if (user === null || user.org_id === null) {
```

Comment:
Warum?

Reply:
Ganz einfach:
Wenn der User keiner Org angehört, dann darf er auch nicht die Routen benutzen, die Org-Spezifisch sind. und das sind alle die "userIsAuthorized" als middleware dazwischen haben.
Andernfalls geht uns dort innendrin immer das Vars.currentOrganization.id kaputt wegen "id of null".